src/Message/Response.php

```diff
@@ -577,6 +577,11 @@ public function getRiskScore()
     public function getInvoiceReferences()
     {
         if (isset($this->data->invoicenum)) {
+            // Vindicia may mess up the field if only one invoice num is returned
```

Comment:
it's actually not really Vindicia's fault, it's the way PHP SOAP parsing works. so probably no need to blame them :-)

Reply:
made the change!